hips/hip-0012.md

```diff
@@ -185,7 +187,7 @@ Cleanup and non-functional changes may proceed while in the planning phase.
 
 #### Feature Development
 
-During this phase, agreed-upon features and changes from the planning stage will be implemented in a development branch of Helm.
```

Comment:
Removed this, as we just discussed that we would use the `main` branch (And I happened to notice this in the HIP while making the other changes). Removed, because I think details such as development branches can be decided outside of the HIP.